README.md

```diff
@@ -61,10 +61,11 @@ Yes, you can enable guest voting by adding this snippet to a mu-plugin:
 Please note that it uses cookies and IP addresses to identify a visitor. This is not as reliable as
 requiring a user to create an account. Cookies can be deleted, and IP addresses can be shared.
 
-### Can comment weight be negative? ###
+### Can I limit comment weight to positive values only? ###
 
-Yes, you can enable negative weight by adding this snippet to a mu-plugin:
-`add_filter( 'hmn_cp_allow_negative_comment_weight', '__return_true' );`
+Yes, you can enable non-negative weights only by adding this snippet to a mu-plugin:
```

Comment:
I'd prefer not to change the default behaviour of the plugin and rather just fix the underlying issue rather than link to a github issue.